UM/Settings/ContainerRegistry.py

```diff
@@ -146,6 +148,13 @@ def findContainers(self, container_type = None, **kwargs):
                             continue
                         except AttributeError:
                             pass
+                    if key == "read_only":
+                        try:
+                            if value != container.isReadOnly():
+                                matches_container = False
+                            continue
+                        except AttributeError:
+                            pass
```

Comment:
I'm personally already developing PTSD just thinking about all of the nasty bugs which will silently escaping logging thanks to the use of `except AttributeError` and `except TypeError`. (This is **terrible** use of `except`.)